File: PHP/Program/SivitasAkademik.php
<?php
include "Mahasiswa.php";

class SivitasAkademik extends Mahasiswa{

	private $univ = '';
	private $email= '';

	public function __construct($univ = '', $email= ''){
        $this->univ = $univ;
        $this->email= $email;
    }

	//getter dan setter
    public function setUniv($univ)
	{
		$this->univ = $univ;
	}

	public function getUniv()
	{
		return $this->univ;
	}

	public function setEmail($email)
	{
		$this->email= $email;
	}

	public function getEmail()
	{
		return $this->email;
	}
}

?>

File: Python/Program/SivitasAkademik.py
from Mahasiswa import Mahasiswa

class SivitasAkademik(Mahasiswa):
	__univ = ''
	__email = ''
	
	def __init__(self, univ = '', email = ''):
		self.__univ = univ
		self.__email = email

	#getter dan setter
	def setUniv(self, univ):
		self.__univ = univ

	def getUniv(self):
		return self.__univ

	def setEmail(self, email):
		self.__email = email

	def getEmail(self):
		return self.__email